lib/get-unified-config.js

```diff
@@ -67,6 +67,7 @@ function getUnifiedConfig () {
     tokenBucketCapacity: {location: 'e', type: 'i', eeid: 'lambda'},
     tokenBucketRate: {location: 'e', type: 'i', eeid: 'lambda'},
     transactionName: {location: 'e', type: 's', eeid: 'lambda'},
+    testKeyWithDefault: {location: 'b', type: 'i', eeid: 'lambda', default: 42},
```

Comment:
Should this be part of test setup, instead of an actually defined setting?

Reply:
ideally it would be part of a test setup, but that would require exposing the configuration object as opposed to keeping it isolated to one module. my thought was that you'd replace this config item with `stdoutConfigNonBlocking` and change the test so it used that. i viewed `testKeyWithDefault` as a placeholder.

if that doesn't make sense to you i can expose the config itself and make those changes.

Reply:
I see, yes that works.